base/lnls-prune-artifacts.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -eu
+
+target_paths=$@
+
+
```

Comment:
I don't think this variable needs to be set; the function call at the end can simply use `$@` itself.

Putting this at the start of the script makes it look like you are going to manipulate the array in the middle of it.